packages/apollo-react-native/src/index.native.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/export */
```

Comment:
We're shadowing exports from the non-native lib here. Not sure if there's a non-explicit way to do this, couldn't get anything to work in my quick test so I just disabled the eslint rule